tests/system/action/user/test_send_invitation_email.py

```diff
@@ -208,7 +208,7 @@ def test_SMTPSenderRefused_not_authenticated(self) -> None:
             )
         self.assert_status_code(response, 200)
         self.assertIn(
-            f"SMTPSenderRefused: (530, b'5.7.0 Authentication required', '{EmailSettings.default_from_email}')",
+            f"Exception: (530, b'5.7.0 Authentication required', '{EmailSettings.default_from_email}') Count 1, Index 0",
```

Comment:
This should not occur. Please catch the SMTPSenderRefused in the inner part and throw it again to be caught in outer asserts. Reason: This occurs in sending email in **connected state**, but the sender is always the same and there is no need to assert each of the mails separately.
You'll stay with the problem of writing a test for this. You could use the AIOHandler in tests/system/action/mail_base.py and raise an Exception for a special address in mail. But that's on your fantasy;-)

Reply:
I have reraised the SMTPSenderRefused Exception. And cleanup the code.